diff -uNr a/src/dde-file-manager-lib/interfaces/dfmcrumbbar.cpp b/src/dde-file-manager-lib/interfaces/dfmcrumbbar.cpp
--- a/src/dde-file-manager-lib/interfaces/dfmcrumbbar.cpp	2022-07-26 09:48:16.000000000 +0800
+++ b/src/dde-file-manager-lib/interfaces/dfmcrumbbar.cpp	2022-08-06 18:12:27.738959866 +0800
@@ -605,8 +605,11 @@
     menu->addAction(copyIcon, QObject::tr("Copy path"), [ = ]() {
         // 如果为保险箱路径则进行路径转换
         QString virtualUrl(url.toString());
-        if (VaultController::isVaultFile(virtualUrl))
+        if (VaultController::isVaultFile(virtualUrl)) {
             virtualUrl = VaultController::localPathToVirtualPath(url.toLocalFile());
+        } else if (url.isLocalFile()) {
+            virtualUrl = url.toLocalFile();
+        }
         QGuiApplication::clipboard()->setText(virtualUrl);
     });
 
