diff -uNr a/src/dde-desktop/presenter/gridmanager.cpp b/src/dde-desktop/presenter/gridmanager.cpp
--- a/src/dde-desktop/presenter/gridmanager.cpp	2022-06-12 18:06:01.000000000 +0800
+++ b/src/dde-desktop/presenter/gridmanager.cpp	2022-06-14 09:08:55.069531038 +0800
@@ -55,17 +55,7 @@
                     << dir.filePath("dde-computer.desktop");
 
         autoArrange = Config::instance()->getConfig(Config::groupGeneral, Config::keyAutoAlign).toBool();
-#ifdef ENABLE_AUTOMERGE  //sp2需求调整，屏蔽自动整理
         autoMerge = Config::instance()->getConfig(Config::groupGeneral, Config::keyAutoMerge, false).toBool();
-#else
-        //取消自动整理
-        autoMerge = Config::instance()->getConfig(Config::groupGeneral, Config::keyAutoMerge, false).toBool();
-        //若用户开启了自动整理，则强制退出
-        if (autoMerge) {
-            autoMerge = false;
-            Config::instance()->setConfig(Config::groupGeneral, Config::keyAutoMerge, false);
-        }
-#endif
 //        auto settings = Config::instance()->settings();
 //        settings->beginGroup(Config::groupGeneral);
 //        autoArrange = settings->value(Config::keyAutoAlign).toBool();
diff -uNr a/src/dde-desktop/view/canvasgridview.cpp b/src/dde-desktop/view/canvasgridview.cpp
--- a/src/dde-desktop/view/canvasgridview.cpp	2022-06-12 18:06:01.000000000 +0800
+++ b/src/dde-desktop/view/canvasgridview.cpp	2022-06-14 09:09:36.549819952 +0800
@@ -2952,7 +2952,6 @@
         }
     });
 
-#ifdef ENABLE_AUTOMERGE  //sp2需求调整，屏蔽自动整理
     connect(this, &CanvasGridView::autoMergeToggled, this, []() {
         bool enable = !GridManager::instance()->autoMerge();
         GridManager::instance()->setAutoMerge(enable);
@@ -2960,7 +2959,6 @@
 
         emit GridManager::instance()->sigSyncOperation(GridManager::soAutoMerge, enable);
     });
-#endif
 
     connect(this, &CanvasGridView::sortRoleChanged,
             Presenter::instance(), &Presenter::onSortRoleChanged);
@@ -3437,7 +3435,6 @@
     iconSizeAction.setMenu(&iconSizeMenu);
     menu->insertAction(pasteAction, &iconSizeAction);
 
-#ifdef ENABLE_AUTOMERGE  //sp2需求调整，屏蔽自动整理
     //自动整理
     QAction menuAutoMerge(menu);
     menuAutoMerge.setText(tr("Auto merge"));
@@ -3448,7 +3445,6 @@
     if (settings.value("auto-merge").toBool()) {
         menu->insertAction(pasteAction, &menuAutoMerge);
     }
-#endif
 
     //自动排序
     QAction autoSort(menu);
