diff -uNr a/src/dde-file-manager-lib/views/computerviewitemdelegate.cpp b/src/dde-file-manager-lib/views/computerviewitemdelegate.cpp
--- a/src/dde-file-manager-lib/views/computerviewitemdelegate.cpp	2022-06-12 18:06:01.000000000 +0800
+++ b/src/dde-file-manager-lib/views/computerviewitemdelegate.cpp	2022-06-14 09:22:26.270030584 +0800
@@ -129,7 +129,7 @@
         painter->drawPixmap(option.rect.x() + leftmargin, option.rect.y() + topmargin, icon.pixmap(iconsize));
 
         painter->setFont(par->font());
-        painter->setPen(qApp->palette().color((option.state & QStyle::StateFlag::State_Selected) ? QPalette::ColorRole::HighlightedText : QPalette::ColorRole::Text));
+        painter->setPen(qApp->palette().color(QPalette::ColorRole::Text));
         painter->drawText(option.rect.x() + (option.rect.width() - fstw) / 2, option.rect.y() + topmargin + iconsize + text_topmargin, elided_text);
         return;
     }
