diff -uNr a/src/dde-file-manager-lib/models/dfmrootfileinfo.cpp b/src/dde-file-manager-lib/models/dfmrootfileinfo.cpp
--- a/src/dde-file-manager-lib/models/dfmrootfileinfo.cpp	2022-06-12 18:06:01.000000000 +0800
+++ b/src/dde-file-manager-lib/models/dfmrootfileinfo.cpp	2022-06-16 10:22:31.260259695 +0800
@@ -166,14 +166,24 @@
     return false; //默认返回false warning项
 }
 
+QHash<QString, QString> suffixCache;
+QRegularExpression suffixRe(".*\\.(.*)$");
+
 QString DFMRootFileInfo::suffix() const
 {
-    QRegularExpression re(".*\\.(.*)$");
-    auto rem = re.match(fileName());
+    auto fname = fileName();
+    auto cdata = suffixCache.find(fname);
+    if (cdata != suffixCache.end()) {
+        return cdata.value();
+    }
+    auto rem = suffixRe.match(fname);
     if (!rem.hasMatch()) {
+        suffixCache.insert(fname, "");
         return "";
     }
-    return rem.captured(1);
+    auto v = rem.captured(1);
+    suffixCache.insert(fname, v);
+    return v;
 }
 
 QString DFMRootFileInfo::fileDisplayName() const
